packages/react-generator/test/generator/expected/appdemo01/src/lowcodeConfig/dataSource.js

```
@@ -0,0 +1,102 @@
+ */
+
+import useHttp from '../http'
+import dataSources from './dataSource.json'
+
+const dataSourceMap = {}
+
+// 暂时使用 eval 解析 JSON 数据里的函数
+const createFn = (fnContent) => {
+  return (...args) => {
+    // eslint-disable-next-line no-eval
+    window.eval('var fn = ' + fnContent)
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)